distributed/utils.py

```diff
@@ -371,17 +371,27 @@ def in_async_call(loop, default=False):
         return False
 
 
-def sync(loop, func, *args, callback_timeout=None, **kwargs):
+def sync(
+    loop: IOLoop,
+    func: Callable[..., Awaitable[T] | asyncio.Future[T]],
```

Comment:
Future[T] is already an Awaitable[T] so this union is redundant

Reply:
Addressed.